src/scores/categorical/contingency_impl.py

```diff
@@ -131,8 +131,9 @@ def base_rate(self) -> xr.DataArray:
             - "False negatives" is the same as "misses".
 
         References:
-            Jolliffe, I.T. and Stephenson, D.B. eds., 2012. Forecast verification: a
-            practitioner's guide in atmospheric science. John Wiley & Sons.
+            Hogan, R. J. & Mason, I. B. (2011). Deterministic forecasts of binary events.
+            In I. T. Jolliffe & D. B. Stephenson (Eds.), Forecast Verification (2nd ed,
+            pp. 39-51). https://doi.org/10.1002/9781119960003.ch3
```

Comment:
```suggestion
            Hogan, R. J. & Mason, I. B. (2011). Deterministic forecasts of binary events.
            In I. T. Jolliffe & D. B. Stephenson (Eds.), Forecast verification: A practitioner's guide in atmospheric science (2nd ed.,
            pp. 39-51). https://doi.org/10.1002/9781119960003.ch3
```
Sorry - this was my error - the citation I provided earlier contained errors. Apologies.